packages/framework/presence/package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "@fluid-experimental/presence",
+	"name": "@fluidframework/presence",
```

Comment:
We also need to ensure this package includes `/alpha` in its `exports`, right? Otherwise, everything gets surfaced as implicitly "public".

Reply:
Will follow similar changes to this: https://github.com/microsoft/FluidFramework/pull/23069

Reply:
Nit: #23069 preserved the (empty) "." entry, but it looks like we removed it here. I don't know what impact this has on tooling, but for consistency it might be worth restoring the "." entry?

(Feel free to close with no action if this was a deliberate decision, just wanted to check 🙂)

Reply:
It was deliberate. There is nothing in public. There is a slight advantage not to have a useless public entry per the bug in VS Code + Intellisence for completing imports. Without a "." entry that has index.js, auto-import will use /alpha in import spec.